### PR TITLE
Add EL7 support

### PR DESCRIPTION
**Pull request by TheDude05**

### CODE REVIEW

File: Dockerfile-el7 [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM centos:centos7
+
+ENV HOME /root
+
+#RUN yum localinstall -y http://mirror.pnl.gov/epel/6/i386/epel-release-6-8.noarch.rpm
```
Comment (dermusikman) [inline Dockerfile-el7]: Comment still here

**Comment by dermusikman** [comment]
Presently failing with the following:
```
[root@ip-172-31-28-38 ~]# yum localinstall -y nginx-1.7.10-2.el6_intox.x86_64.rpm
Loaded plugins: fastestmirror
Examining nginx-1.7.10-2.el6_intox.x86_64.rpm: nginx-1.7.10-2.el6_intox.x86_64
Marking nginx-1.7.10-2.el6_intox.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package nginx.x86_64 0:1.7.10-2.el6_intox will be installed
--> Processing Dependency: GeoIP for package: nginx-1.7.10-2.el6_intox.x86_64
Loading mirror speeds from cached hostfile
 * base: mirror.hmc.edu
 * epel: mirror.prgmr.com
 * extras: centos.sonn.com
 * updates: repos.dfw.quadranet.com
--> Processing Dependency: gd for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libxslt.so.1(LIBXML2_1.0.11)(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libxslt.so.1(LIBXML2_1.0.18)(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: perl(:MODULE_COMPAT_5.10.1) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libGeoIP.so.1()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libexslt.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libgd.so.2()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libpcre.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libxslt.so.1()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Running transaction check
---> Package GeoIP.x86_64 0:1.5.0-9.el7 will be installed
---> Package gd.x86_64 0:2.0.35-26.el7 will be installed
--> Processing Dependency: libpng15.so.15(PNG15_0)(64bit) for package: gd-2.0.35-26.el7.x86_64
--> Processing Dependency: libpng15.so.15()(64bit) for package: gd-2.0.35-26.el7.x86_64
--> Processing Dependency: libfontconfig.so.1()(64bit) for package: gd-2.0.35-26.el7.x86_64
--> Processing Dependency: libXpm.so.4()(64bit) for package: gd-2.0.35-26.el7.x86_64
--> Processing Dependency: libX11.so.6()(64bit) for package: gd-2.0.35-26.el7.x86_64
---> Package libxslt.x86_64 0:1.1.28-5.el7 will be installed
---> Package nginx.x86_64 0:1.7.10-2.el6_intox will be installed
--> Processing Dependency: perl(:MODULE_COMPAT_5.10.1) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libpcre.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Running transaction check
---> Package fontconfig.x86_64 0:2.10.95-7.el7 will be installed
--> Processing Dependency: fontpackages-filesystem for package: fontconfig-2.10.95-7.el7.x86_64
---> Package libX11.x86_64 0:1.6.0-2.1.el7 will be installed
--> Processing Dependency: libX11-common = 1.6.0-2.1.el7 for package: libX11-1.6.0-2.1.el7.x86_64
--> Processing Dependency: libxcb.so.1()(64bit) for package: libX11-1.6.0-2.1.el7.x86_64
---> Package libXpm.x86_64 0:3.5.10-5.1.el7 will be installed
---> Package libpng.x86_64 2:1.5.13-5.el7 will be installed
---> Package nginx.x86_64 0:1.7.10-2.el6_intox will be installed
--> Processing Dependency: perl(:MODULE_COMPAT_5.10.1) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libpcre.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Running transaction check
---> Package fontpackages-filesystem.noarch 0:1.44-8.el7 will be installed
---> Package libX11-common.noarch 0:1.6.0-2.1.el7 will be installed
---> Package libxcb.x86_64 0:1.9-5.el7 will be installed
--> Processing Dependency: libXau.so.6()(64bit) for package: libxcb-1.9-5.el7.x86_64
---> Package nginx.x86_64 0:1.7.10-2.el6_intox will be installed
--> Processing Dependency: perl(:MODULE_COMPAT_5.10.1) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libpcre.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Running transaction check
---> Package libXau.x86_64 0:1.0.8-2.1.el7 will be installed
---> Package nginx.x86_64 0:1.7.10-2.el6_intox will be installed
--> Processing Dependency: perl(:MODULE_COMPAT_5.10.1) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Processing Dependency: libpcre.so.0()(64bit) for package: nginx-1.7.10-2.el6_intox.x86_64
--> Finished Dependency Resolution
Error: Package: nginx-1.7.10-2.el6_intox.x86_64 (/nginx-1.7.10-2.el6_intox.x86_64)
           Requires: perl(:MODULE_COMPAT_5.10.1)
Error: Package: nginx-1.7.10-2.el6_intox.x86_64 (/nginx-1.7.10-2.el6_intox.x86_64)
           Requires: libpcre.so.0()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@ip-172-31-28-38 ~]# yum repolist
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.hmc.edu
 * epel: mirror.prgmr.com
 * extras: centos.sonn.com
 * updates: repos.dfw.quadranet.com
repo id                                                                repo name                                                                                            status
base/7/x86_64                                                          CentOS-7 - Base                                                                                      8,465
epel/x86_64                                                            Extra Packages for Enterprise Linux 7 - x86_64                                                       7,378
extras/7/x86_64                                                        CentOS-7 - Extras                                                                                      104
updates/7/x86_64                                                       CentOS-7 - Updates                                                                                   1,723
repolist: 17,670
```

**Comment by TheDude05** [comment]
As discussed in person, the `default` build target was not including 'centos7'. Adjusted

**Comment by dermusikman** [comment]
Seems to work. Merging/deleting.

**Pull request merged**